crates/version/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "calimero-version"
+version = "0.5.0"
+edition = "2021"
+
+[dependencies]
+once_cell = "1.19.0" 
```

Comment:
[`LazyLock`](https://doc.rust-lang.org/std/sync/struct.LazyLock.html) has been stable since rust 1.80, we don't need this dependency